devs/docs/architecture.rst

```diff
@@ -53,26 +53,19 @@ An advantage of the Postgres protocol in CrateDB is that you can use it also for
 applications which were originally built to communicate with Postgres. You may
 also use Postgres tools like `psql` with CrateDB.
 
-The enterprise version of CrateDB also contains support for ingesting data with
-the MQTT protocol.
-
 Class entry points:
 
 - `CrateRestMainAction`_
 - `PostgresWireProtocol`_
-- `MqttNettyHandler`_
 
 Transport
 ..........
 
 Transport denotes the communication between nodes in a CrateDB cluster. CrateDB
-uses Elasticsearch to manage the cluster state and to transfer data between
-cluster nodes. This includes node discovery, partitioning of data, recovery, and
-replication.
+components used to manage the cluster state and to transfer data between
+cluster nodes are based on Elasticsearch code. This includes node discovery,
```

Comment:
this paragraph doesn't make sense

Reply:
Should I just remove the whole paragraph except for the first sentence? I don't think pointing out the code origins matters. We'll soon have the submodule gone and then it is just different modules

Reply:
@andreidan okay, pushed a fixup